hack/dind-cluster.sh

```diff
@@ -220,6 +220,9 @@ function wait-for-cluster() {
   local oc
   oc="$(os::build::find-binary oc)"
 
+  # wait for healthz to report ok before trying to get nodes
```

Comment:
> I don't see why this change would be necessary. Consider removing.

I think its reasonable to wait until the API server is ready before making real requests to it. This mirrors what other e2e tests do.